### ISSUE TITLE
Accessibility (WCAG level A) Skipped header level on site's front page.

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1181]
1.  Go to http://nuget.org/ and view source
2.  Scrutinize how we're using h1..h6 tags to organize headers.

Result:  We use h1 and h3 elements in the front page, but no h2 elements.  
Expected:  We shouldn't skip levels when using the h1..h6 tags.  In other words, if we're using an h3 tag, that means there should be an h1 and h2 tag somewhere else on the page.  In this case, I'd expect us to use h1 and h2 tags and re-style the tags as necessary to achieve the visual effect we want.

**Comment by johnataylor** [comment]
We have a new gallery design.
